doc/command_ref.rst

```diff
@@ -563,17 +563,17 @@ different distribution release versions.
 .. _deplist_command-label:
 
 ---------------
-Deplist command
+Deplist Command
 ---------------
 
 ``dnf [options] deplist [<select-options>] [<query-options>] [<package-spec>]``
     Alias for :ref:`dnf repoquery --deplist <deplist_option-label>`.
 
 .. _distro_sync_command-label:
 
--------------------
-Distro-Sync command
--------------------
+----------------------------------
+Distro-Sync Command (alias: dsync)
+----------------------------------
```

Comment:
Hmm... I think the aliases shouldn't really be part of the heading... How about:
```
-------------------
Distro-Sync command
-------------------

Aliases: dsync

...
```

I actually find it a bit confusing myself that the commands in the headings are Capitalized while the commands themselves are lowercase...

Reply:
I could go either way, but in both cases, I wouldn't list all aliases, just the short ones. Otherwise, there are commands such as distro-sync: ('distro-sync', 'distrosync', 'distribution-synchronization', 'dsync')

Reply:
> I could go either way, ...

By either way you mean either your current approach or my suggestion?

> ... but in both cases, I wouldn't list all aliases, just the short ones. Otherwise, there are commands such as distro-sync: ('distro-sync', 'distrosync', 'distribution-synchronization', 'dsync')

Hmm... what is the difference then? Are those long variations deprecated? If so I think it would be a good idea to mark those as deprecated in some way, either user-facing (in the documentation) or even just in the code. Just to make it clear which aliases are there for compatibility reasons and we would like to remove them eventually.

Reply:
> By either way you mean either your current approach or my suggestion?

Yes.

> > ... but in both cases, I wouldn't list all aliases, just the short ones. Otherwise, there are commands such as distro-sync: ('distro-sync', 'distrosync', 'distribution-synchronization', 'dsync')
> 
> Hmm... what is the difference then? Are those long variations deprecated? If so I think it would be a good idea to mark those as deprecated in some way, either user-facing (in the documentation) or even just in the code. Just to make it clear which aliases are there for compatibility reasons and we would like to remove them eventually.

Some of them are deprecated and are already documented in separate section (e.g. `distribution-synchronization`), some are not documented, so I don't know if we want to deprecate them.

I see few options:

1. Document all aliases in separate sections
2. Document all aliases in the same section as the main command
3. Document all aliases in the same section as the main command, but deprecated aliases in separate sections

Ad 1 - I don't like this, since it would also mean to create e.g. section for alias "in" referencing the install command.

Ad 2 - The only disadvantage I see is that user won't be able to find a deprecated command alphabetically. It could look like this for example for distro-sync command:
```
Aliases: dsync
Deprecated aliases: distrosync, distribution-synchronization 
```

Ad 3 - A bit inconsistent, but would require the least amount of change (just to add few undocumented aliases).

Any thoughts?

Reply:
Option 2 is what I had in mind and I think it's the best one... Not sure exactly how you mean the user would be searching alphabetically for the alias, as it doesn't seem to be an issue to me... Since they are deprecated these aliases are second-class citizens already :slightly_smiling_face: